buildSrc/build.gradle.kts

```diff
@@ -3,7 +3,7 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 buildscript {
 
     val pluginsExperiments by extra(
```

Comment:
:x: Rename `pluginsExperiments` to something closer to the new dependency name.

Reply:
Fixed